includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -527,6 +530,7 @@ class AMP_Allowed_Tags_Generated {
 			'hr',
 			'i',
 			'image',
+			'img',
```

Comment:
I just tried adding an `img` to an `amp-story-page-attachment` and I get a validation error. See [playground](https://playground.amp.dev/#share=PCFET0NUWVBFIGh0bWw+CjxodG1sIGFtcCBsYW5nPSJlbiI+CjxoZWFkPgogIDxtZXRhIGNoYXJzZXQ9InV0Zi04IiAvPgogIDxzY3JpcHQgYXN5bmMgc3JjPSJodHRwczovL2Nkbi5hbXBwcm9qZWN0Lm9yZy92MC5qcyI+PC9zY3JpcHQ+CiAgPHNjcmlwdCBhc3luYyBjdXN0b20tZWxlbWVudD0iYW1wLXlvdXR1YmUiIHNyYz0iaHR0cHM6Ly9jZG4uYW1wcHJvamVjdC5vcmcvdjAvYW1wLXlvdXR1YmUtMC4xLmpzIj48L3NjcmlwdD4KICA8c2NyaXB0IGFzeW5jIGN1c3RvbS1lbGVtZW50PSJhbXAtdHdpdHRlciIgc3JjPSJodHRwczovL2Nkbi5hbXBwcm9qZWN0Lm9yZy92MC9hbXAtdHdpdHRlci0wLjEuanMiPjwvc2NyaXB0PgogIDxzY3JpcHQgYXN5bmMgY3VzdG9tLWVsZW1lbnQ9ImFtcC1zdG9yeSIgc3JjPSJodHRwczovL2Nkbi5hbXBwcm9qZWN0Lm9yZy92MC9hbXAtc3RvcnktMS4wLmpzIj48L3NjcmlwdD4KICA8dGl0bGU+SGVsbG8sIGFtcC1zdG9yeTwvdGl0bGU+CiAgPGxpbmsgcmVsPSJjYW5vbmljYWwiIGhyZWY9Imh0dHA6Ly9leGFtcGxlLmFtcHByb2plY3Qub3JnL215LXN0b3J5Lmh0bWwiIC8+CiAgPG1ldGEgbmFtZT0idmlld3BvcnQiIGNvbnRlbnQ9IndpZHRoPWRldmljZS13aWR0aCxtaW5pbXVtLXNjYWxlPTEsaW5pdGlhbC1zY2FsZT0xIiAvPgogIDxzdHlsZSBhbXAtYm9pbGVycGxhdGU+CiAgICAgIGJvZHkgewogICAgICAgIC13ZWJraXQtYW5pbWF0aW9uOiAtYW1wLXN0YXJ0IDhzIHN0ZXBzKDEsIGVuZCkgMHMgMSBub3JtYWwgYm90aDsKICAgICAgICAtbW96LWFuaW1hdGlvbjogLWFtcC1zdGFydCA4cyBzdGVwcygxLCBlbmQpIDBzIDEgbm9ybWFsIGJvdGg7CiAgICAgICAgLW1zLWFuaW1hdGlvbjogLWFtcC1zdGFydCA4cyBzdGVwcygxLCBlbmQpIDBzIDEgbm9ybWFsIGJvdGg7CiAgICAgICAgYW5pbWF0aW9uOiAtYW1wLXN0YXJ0IDhzIHN0ZXBzKDEsIGVuZCkgMHMgMSBub3JtYWwgYm90aDsKICAgICAgfQogICAgICBALXdlYmtpdC1rZXlmcmFtZXMgLWFtcC1zdGFydCB7CiAgICAgICAgZnJvbSB7CiAgICAgICAgICB2aXNpYmlsaXR5OiBoaWRkZW47CiAgICAgICAgfQogICAgICAgIHRvIHsKICAgICAgICAgIHZpc2liaWxpdHk6IHZpc2libGU7CiAgICAgICAgfQogICAgICB9CiAgICAgIEAtbW96LWtleWZyYW1lcyAtYW1wLXN0YXJ0IHsKICAgICAgICBmcm9tIHsKICAgICAgICAgIHZpc2liaWxpdHk6IGhpZGRlbjsKICAgICAgICB9CiAgICAgICAgdG8gewogICAgICAgICAgdmlzaWJpbGl0eTogdmlzaWJsZTsKICAgICAgICB9CiAgICAgIH0KICAgICAgQC1tcy1rZXlmcmFtZXMgLWFtcC1zdGFydCB7CiAgICAgICAgZnJvbSB7CiAgICAgICAgICB2aXNpYmlsaXR5OiBoaWRkZW47CiAgICAgICAgfQogICAgICAgIHRvIHsKICAgICAgICAgIHZpc2liaWxpdHk6IHZpc2libGU7CiAgICAgICAgfQogICAgICB9CiAgICAgIEAtby1rZXlmcmFtZXMgLWFtcC1zdGFydCB7CiAgICAgICAgZnJvbSB7CiAgICAgICAgICB2aXNpYmlsaXR5OiBoaWRkZW47CiAgICAgICAgfQogICAgICAgIHRvIHsKICAgICAgICAgIHZpc2liaWxpdHk6IHZpc2libGU7CiAgICAgICAgfQogICAgICB9CiAgICAgIEBrZXlmcmFtZXMgLWFtcC1zdGFydCB7CiAgICAgICAgZnJvbSB7CiAgICAgICAgICB2aXNpYmlsaXR5OiBoaWRkZW47CiAgICAgICAgfQogICAgICAgIHRvIHsKICAgICAgICAgIHZpc2liaWxpdHk6IHZpc2libGU7CiAgICAgICAgfQogICAgICB9CiAgICA8L3N0eWxlPgogIDxub3NjcmlwdAogICAgICA+PHN0eWxlIGFtcC1ib2lsZXJwbGF0ZT4KICAgICAgICBib2R5IHsKICAgICAgICAgIC13ZWJraXQtYW5pbWF0aW9uOiBub25lOwogICAgICAgICAgLW1vei1hbmltYXRpb246IG5vbmU7CiAgICAgICAgICAtbXMtYW5pbWF0aW9uOiBub25lOwogICAgICAgICAgYW5pbWF0aW9uOiBub25lOwogICAgICAgIH0KICAgICAgPC9zdHlsZT48L25vc2NyaXB0Pgo8L2hlYWQ+Cjxib2R5PgogIDxhbXAtc3Rvcnkgc3RhbmRhbG9uZSB0aXRsZT0iSGVsbG8gU3RvcnkiIHB1Ymxpc2hlcj0iVGhlIEFNUCBUZWFtIiBwdWJsaXNoZXItbG9nby1zcmM9Imh0dHBzOi8vZXhhbXBsZS5jb20vbG9nby8xeDEucG5nIiBwb3N0ZXItcG9ydHJhaXQtc3JjPSJodHRwczovL2V4YW1wbGUuY29tL215LXN0b3J5L3Bvc3Rlci8zeDQuanBnIj4KICAgIDxhbXAtc3RvcnktcGFnZSBpZD0ibXktZmlyc3QtcGFnZSI+CiAgICAgIDxhbXAtc3RvcnktZ3JpZC1sYXllciB0ZW1wbGF0ZT0iZmlsbCI+CiAgICAgICAgPGFtcC1pbWcgc3JjPSJodHRwczovL2V4YW1wbGUuYW1wcHJvamVjdC5vcmcvaGVsbG93b3JsZC9iZzEuanBnIiB3aWR0aD0iOTAwIiBoZWlnaHQ9IjE2MDAiIGFsdD0iIj4KICAgICAgICA8L2FtcC1pbWc+CiAgICAgIDwvYW1wLXN0b3J5LWdyaWQtbGF5ZXI+CiAgICAgIDxhbXAtc3RvcnktZ3JpZC1sYXllciB0ZW1wbGF0ZT0idmVydGljYWwiPgogICAgICAgIDxoMT5IZWxsbywgYW1wLXN0b3J5ITwvaDE+CiAgICAgIDwvYW1wLXN0b3J5LWdyaWQtbGF5ZXI+CiAgICA8L2FtcC1zdG9yeS1wYWdlPgoKICAgIDxhbXAtc3RvcnktcGFnZSBpZD0icGFnZS1hdHRhY2htZW50LWV4YW1wbGUiPgogICAgICA8YW1wLXN0b3J5LWdyaWQtbGF5ZXIgdGVtcGxhdGU9ImZpbGwiPgogICAgICAgIDxhbXAtaW1nIHNyYz0iaHR0cHM6Ly9leGFtcGxlLmFtcHByb2plY3Qub3JnL2hlbGxvd29ybGQvYmcxLmpwZyIgd2lkdGg9IjkwMCIgaGVpZ2h0PSIxNjAwIj4KICAgICAgPC9hbXAtc3RvcnktZ3JpZC1sYXllcj4KICAgICAgPGFtcC1zdG9yeS1wYWdlLWF0dGFjaG1lbnQgbGF5b3V0PSJub2Rpc3BsYXkiPgogICAgICAgIDxoMT5NeSBJbWFnZTwvaDE+CiAgICAgICAgPGltZyBkZWNvZGluZz0iYXN5bmMiIGFsdD0iIiBzcmM9Imh0dHBzOi8vYmxvZy5hbXAuZGV2L3dwLWNvbnRlbnQvdXBsb2Fkcy8yMDIxLzA5L2ltYWdlNC0xMDI0eDUxMi5qcGciIHNyY3NldD0iaHR0cHM6Ly9ibG9nLmFtcC5kZXYvd3AtY29udGVudC91cGxvYWRzLzIwMjEvMDkvaW1hZ2U0LTEwMjR4NTEyLmpwZyAxMDI0dywgaHR0cHM6Ly9ibG9nLmFtcC5kZXYvd3AtY29udGVudC91cGxvYWRzLzIwMjEvMDkvaW1hZ2U0LTMwMHgxNTAuanBnIDMwMHcsIGh0dHBzOi8vYmxvZy5hbXAuZGV2L3dwLWNvbnRlbnQvdXBsb2Fkcy8yMDIxLzA5L2ltYWdlNC03Njh4Mzg0LmpwZyA3Njh3LCBodHRwczovL2Jsb2cuYW1wLmRldi93cC1jb250ZW50L3VwbG9hZHMvMjAyMS8wOS9pbWFnZTQtMTUzNng3NjguanBnIDE1MzZ3LCBodHRwczovL2Jsb2cuYW1wLmRldi93cC1jb250ZW50L3VwbG9hZHMvMjAyMS8wOS9pbWFnZTQuanBnIDE5OTl3Ij4KICAgICAgPC9hbXAtc3RvcnktcGFnZS1hdHRhY2htZW50PgogICAgPC9hbXAtc3RvcnktcGFnZT4KICAgIDxhbXAtc3RvcnktcGFnZSBpZD0ibXktc2Vjb25kLXBhZ2UiPgogICAgICA8YW1wLXN0b3J5LWdyaWQtbGF5ZXIgdGVtcGxhdGU9ImZpbGwiPgogICAgICAgIDxhbXAtaW1nIHNyYz0iaHR0cHM6Ly9leGFtcGxlLmFtcHByb2plY3Qub3JnL2hlbGxvd29ybGQvYmcyLmdpZiIgd2lkdGg9IjkwMCIgaGVpZ2h0PSIxNjAwIiBhbHQ9IiI+CiAgICAgICAgPC9hbXAtaW1nPgogICAgICA8L2FtcC1zdG9yeS1ncmlkLWxheWVyPgogICAgICA8YW1wLXN0b3J5LWdyaWQtbGF5ZXIgdGVtcGxhdGU9InZlcnRpY2FsIj4KICAgICAgICA8aDE+VGhlIEVuZDwvaDE+CiAgICAgIDwvYW1wLXN0b3J5LWdyaWQtbGF5ZXI+CiAgICA8L2FtcC1zdG9yeS1wYWdlPgogIDwvYW1wLXN0b3J5Pgo8L2JvZHk+CjwvaHRtbD4=).

It looks like perhaps the AMP Story components haven't been updated to support native images?

Reply:
See spec for `amp-story-page-attachment`: https://github.com/ampproject/amphtml/blob/4ce3cd79520dbeaf5ed5364cbff58d3d71dee40e/extensions/amp-story/validator-amp-story.protoascii#L1056-L1256